std/math/hardware.d

```diff
@@ -1049,6 +1059,11 @@ private:
             {
                 asm pure nothrow @nogc { "frcsr %0" : "=r" (cont); }
             }
+            else version (LoongArch_Any)
+            {
+                asm pure nothrow @nogc { "movfcsr2gr %0, $fcsr0" : "=r" (cont); }
+                cont &= (roundingMask | allExceptions);
```

Comment:
Hi, thanks for the PRs! Looks reasonable to me, except for this line here - is the masking really required here for this internal function? We don't mask for the other ISAs.

Reply:
It seems so. This is copied from https://github.com/ldc-developers/phobos/pull/73/files#diff-a76c79337d2c4e1daf6668e751145248ef1b49cea7f56ac1fbb39e4d6e0b05acL1093

Thanks.

Reply:
Oh I see, some ugly duplication - about time to finally upstream this stuff.